src/utils/neuroglancer.js

```diff
@@ -0,0 +1,46 @@
+// FIXME: Temporary hack for inferring the type of a layer without an explicit type.
+function inferredLayerType(layer) {
+  if (layer.name.includes('segmentation')) {
+    return 'segmentation';
+  }
+  if (layer.location.includes('segmentation')) {
+    return 'segmentation';
+  }
+  return undefined;
+}
+
+/* eslint-disable-next-line  import/prefer-default-export */
+export function addLayersFromDataset(layers, dataset, inferringType) {
```

Comment:
This function has side effects, which can be confusing. How about changing it to

const updateLayers = addLayersFromDataset(layers, dataset, inferringType);

Then we know explicitly that the layers have been updated by this function.

Reply:
This looks good to me. I'll change it to remove side effects.